packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadListActionsRequired.tsx

```diff
@@ -10,9 +13,59 @@ import { Box } from '@mui/material'
 
 export type DownloadListActionsRequiredProps = {
   /** Invoked when a user clicks "View Sharing Settings" for a set of files that require the Download permission*/
-  onViewSharingSettingsClicked: (benefactorId: string) => void
+  onViewSharingSettingsClicked?: (benefactorId: string) => void
 }
 
+export type ActionRequiredParams = {
+  action: Action
+  count?: number
+  /** Invoked when a user clicks "View Sharing Settings" for a set of files that require the Download permission*/
+  onViewSharingSettingsClicked?: (benefactorId: string) => void
+}
+
+/**
+ * Returns rendering for the Action and optional count.
+ *
+ * @param {Action} action action being rendered
+ * @param {number} count number of files associated to the Action (optional)
+ */
+export const renderActionRequired = (params: ActionRequiredParams) => {
```

Comment:
Thanks for pulling this out, could we also give it a `PascalCase` name and call it with JSX? I know `ActionRequiredCard` is taken, maybe this could be something like `ActionRequiredListItem`?